cpp/include/patterns/copy_v_transform_reduce_key_aggregated_out_nbr.cuh

```diff
@@ -66,7 +66,8 @@ __global__ void for_all_major_for_all_nbr_low_degree(
     thrust::tie(indices, weights, local_degree) =
       matrix_partition.get_local_edges(static_cast<vertex_t>(major_offset));
     if (local_degree > 0) {
-      auto local_offset    = matrix_partition.get_local_offset(major_offset);
+      auto local_offset = matrix_partition.get_local_offset(major_offset) -
+                          matrix_partition.get_local_offset(major_first);
```

Comment:
Are we sure this change is correct?  In my debugging on 2 GPUs, I observed that local_offset on GPU 1 was negative.  Reverting this to the original line generated correct results in my MNMG Louvain tests while using this change resulted in array references being incorrect and sometimes out of bounds.

Reply:
Oh, yes... you're correct. Not sure why I made this change :-( I will undo the "en"bugging :-(

Reply:
Done.